tardis/montecarlo/montecarlo_numba/single_packet_loop.py

```diff
@@ -67,12 +70,8 @@ def single_packet_loop(
         r_packet, vpacket_collection, numba_model, numba_plasma
     )
 
-    if mc_tracker.DEBUG_MODE:
-        r_packet_track_nu = [r_packet.nu]
```

Comment:
Try using the numba TypedList instead of default python lists

Reply:
Sorry, I didn't get where this needs to be incorporated 😓